accumulo/src/main/scala/geotrellis/spark/io/accumulo/AccumuloRDDWriter.scala

```diff
@@ -33,8 +33,8 @@ object AccumuloRDDWriter {
         // Map and sort first, so that partitioner is carried over to groupByKey
         raster
           .map { case (key, value) => (encodeKey(key), (key, value)) }
-          .sortByKey()
           .groupByKey()
+          .sortByKey()
```

Comment:
This actually ruins the optimization that the write strategy `requiresSort` was going for, so this sort should go back into the write strategy (since what I was stupidly going for was to carry the partitioner over from the sort to the group so I wouldn't require a shuffle). I'm going to make that change now.